rasa_core/train.py

```diff
@@ -30,7 +30,7 @@ def create_argument_parser():
 
     # either the user can pass in a story file, or the data will get
     # downloaded from a url
-    group = parser.add_mutually_exclusive_group(required=True)
+    group = parser.add_mutually_exclusive_group(required=False)
```

Comment:
how about keeping the exclusive group with required and adding the `--core` to that group - so either you need to supply training data or a model